mass_update_incidents/mass_update_incidents.py

```diff
@@ -43,72 +40,176 @@ def mass_update_incidents(args):
         sinceuntil = args.date_range.split(',')
         if len(sinceuntil) != 2:
             raise ValueError("Date range must be two ISO8601-formatted time "
-                "stamps separated by a comma.")
+                             "stamps separated by a comma.")
         PARAMETERS['since'] = sinceuntil[0]
         PARAMETERS['until'] = sinceuntil[1]
-        print("Getting incidents for date range: "+" to ".join(sinceuntil))
+        print("Getting incidents for date range: " + " to ".join(sinceuntil))
     else:
         PARAMETERS['date_range'] = 'all'
         print("Getting incidents of all time")
-    print("Parameters: "+str(PARAMETERS))
+    print("Parameters: " + str(PARAMETERS))
```

Comment:
Move this line down before line 56. Right now it omits the incident_id parameters.